Cesium3DTilesSelection/src/Tileset.cpp

```diff
@@ -376,7 +376,7 @@ uint32_t Tileset::computeLoadProgress() noexcept {
   uint32_t totalNum = this->_loadedTilesCount + inProgressSum;
   float_t percentage = static_cast<float>(this->_loadedTilesCount) /
```

Comment:
```suggestion
  float percentage = static_cast<float>(this->_loadedTilesCount) /
```

Might as well change it here too.